cpp/farm_ng/core/logging/logger.cpp

```diff
@@ -51,6 +51,7 @@ void StreamLogger::writeHeader(
           fmt::arg("text", header_text),
           fmt::arg("file", file),
           fmt::arg("line", line),
+          fmt::arg("filename", std::filesystem::path(file).filename()),
```

Comment:
let's give it a single parent path for context as well

Reply:
Done